client/App.jsx

```diff
@@ -44,12 +45,13 @@ class App extends React.Component {
     } else if (view === 'limit') {
       return <LimitOrder companies={this.state.companyData}/>
     } else if(view ==='stoploss') {
-      return <StopLossOrder/>
+      return <StopLossOrder companies={this.state.companyData}/>
```

Comment:
Airbnb eslint would prevent you from applying this as: this.state. 
Try deconstructing properties of state before you use them. 😄